util/downloadBuild.py

```diff
@@ -68,7 +68,7 @@ def parseOptions():
 
     parser.set_defaults(
         compileType='dbg',
-        downloadFolder=os.getcwdu(),
+        downloadFolder=os.getcwd().decode("utf-8", errors="replace") if sys.version_info.major == 2 else os.getcwd(),
```

Comment:
Why use `os.getcwd().decode("utf-8")` over `os.getcwdu()` in Python 2? ntfs for example is UTF-16.

For Windows there is an actual `getcwdw()` C function returning mbcs that `getcwdu()` uses under the hood. Otherwise it does basically `getcwd().decode(sys.getfilesystemencoding())` (with default `errors="strict"`).

I'd rather use `os.getcwdu() if sys.version_info.major == 2 else os.getcwd()`and disable the pylint errors.